pubspec.lock

```diff
@@ -967,7 +967,7 @@ packages:
     dependency: transitive
     description: flutter
     source: sdk
-    version: "0.0.99"
+    version: "0.0.0"
```

Comment:
This happened to me too. Not sure why the version resets to "0.0.0" in this update.

Reply:
Yeah, me neither.

Reply:
Looks like that's for the package `sky_engine`. ("Sky" is an early name for Flutter, so that package name basically means "the Flutter engine".)

Searching in the flutter/engine repo with `git log -p -S '0.0.99'` finds https://github.com/flutter/engine/commit/72a64e4e4a1825c3601afe4be05f0c1ba843ea67 which seems to have made that change, deleting a line `version: 0.0.99` from the package's pubspec.yaml.

It looks like the change wasn't particularly discussed. It happened sort of in passing, while deleting the deprecated `author` field and the `homepage` which had an `http:` URL:
https://github.com/flutter/flutter/issues/152636
But presumably it's fine. That version number was last bumped in 2016:
https://github.com/flutter/engine/commit/139bdb96422003f3b931a9dfbbbe2479317bb43d
Given that apparently nothing broke from dropping it, it doesn't seem to be needed.